### PR TITLE
toggle for tetris bags

### PR DESCRIPTION
**Pull request by Seris02**
## About The Pull Request

title

## Why It's Good For The Game

it looks like fucking shit and most people aren't going to like it

## Changelog
:cl:
add: toggle for tetris bags
/:cl:

### CODE REVIEW

**Comment by Arturlang** [comment]
This seems abusable..

**Comment by Ghommie** [comment]
The feature is currently undergoing some bugfixing by kevinz, and personally I don't think this is a good way to fix those issues.

**Comment by Seris02** [comment]
he gave me his blessings to make it, all it does it alter how the bags are viewed by the person, as far as I know

File: modular_citadel/code/modules/client/preferences.dm [hunk 1]
```diff
@@ -16,6 +16,7 @@
 	var/widescreenpref = TRUE
 	var/autostand = TRUE
 	var/auto_ooc = FALSE
+	var/no_tetris_storage = FALSE
```
Comment (Ghommie) [inline modular_citadel/code/modules/client/preferences.dm]: No, move it away from modular_file, I'm still planning on deconstructing that ruin of a folder.

**Review by Ghommie — CHANGES_REQUESTED** [review]

**Review by Ghommie — APPROVED** [review]

**Pull request merged**